packages/prosess-beregningsgrunnlag/src/components/selvstendigNaeringsdrivende/AvvikopplysningerSN.jsx

```diff
@@ -4,71 +4,63 @@ import { FormattedMessage } from 'react-intl';
 import { Normaltekst } from 'nav-frontend-typografi';
 import { Column } from 'nav-frontend-grid';
 import aktivitetStatus from '@fpsak-frontend/kodeverk/src/aktivitetStatus';
+import sammenligningType from '@fpsak-frontend/kodeverk/src/sammenligningType';
 import { FlexRow } from '@fpsak-frontend/shared-components';
 import AvvikopplysningerATFLSN from '../fellesPaneler/AvvikopplysningerATFLSN';
 
+const ingenAvviksvurdering = (forklarendeTekst) => (
+  <FlexRow>
+    <Column xs="12">
+      <Normaltekst>
+        <FormattedMessage id={forklarendeTekst} />
+      </Normaltekst>
+    </Column>
+  </FlexRow>
+);
+
+const nyIArbeidslivet = () => (ingenAvviksvurdering('Beregningsgrunnlag.Avvikssopplysninger.SN.NyIArbeidslivet'));
+const ingenVarigEndringEllerNyoppstartet = () => (ingenAvviksvurdering('Beregningsgrunnlag.Avvikssopplysninger.SN.IkkeVarigEndring'));
```

Comment:
Trur kanskje eg hadde foretrukket om desse var skrevet som konstanter og ikkje metoder:
NY
```suggestion
const NY_I_ARBEIDSLIVET_TEKST = (ingenAvviksvurdering('Beregningsgrunnlag.Avikssopplysninger.SN.NyIArbeidslivet'));
const INGEN_VARIG_ENDRING_ELLER_NYOPPSTARTET_TEKST =  (ingenAvviksvurdering('Beregningsgrunnlag.Avikssopplysninger.SN.IkkeVarigEndring'));
```

Reply:
Det fine med metoder er du kan ha forklarende metodenavn og ikke litt kryptiske string-koder. Men ser poenget og har endret de